Sources/Controllers/2.0/Settings/OAImportCompleteViewController.mm

```diff
@@ -137,7 +137,7 @@ - (void) generateData
         [_data addObject: @{
             @"label": OALocalizedString(@"configure_screen_quick_action"),
             @"iconName": @"ic_custom_quick_action",
-            @"count": [NSString stringWithFormat:@"%ld", profilesCount],
+            @"count": [NSString stringWithFormat:@"%ld",(long)actionsCount],
```

Comment:
Remove (long). NSInteger **is** long https://developer.apple.com/documentation/objectivec/nsinteger?language=objc